src/app/tests/suites/certification/Test_TC_SC_5_2.yaml

```diff
@@ -220,7 +220,37 @@ tests:
                 value: ""
 
     - label:
-          "Step 9: TH removes the Group key set that was added by sending a
+          "Step 9: TH sends a RemoveGroup Command to the Groups cluster with the
```

Comment:
So this test presupposes a groups cluster, but is run against all commissionees.

Can you check with Gibran? He is supposed to be looking at the groups tests and the PICS markings to understand if they are correct and I can guarantee you that this one is not.